src/index.html

```diff
@@ -4,24 +4,365 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
     />
     <title>Catalog</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    />
     <link
       rel="preconnect"
-      href="https://fonts.gstatic.com"
+      href="https://fonts.googleapis.com"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
     />
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
       rel="stylesheet"
-      href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          class="logo_img"
+          src="./images/logo.png"
+          alt="moyo_logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
```

Comment:
Let's use correct BEM naming: if `nav` is a BEM block, we should name it `nav__item`. Same for nav__list, nav__link